features/edit_destroy.feature

```diff
@@ -26,38 +22,79 @@ Background: Logged in
     | 4  | user4 |                 | test3@user.com |
     | 5  | user5 |                 | test@user.com |
     | 6  | user6 |                 | test@user.com |
-
-  And I'm logged in on the orgs page
-  And I am logged in as a "staff"
-  And I follow "Apps"
-  And I follow "app1"
   And the following engagements exist:
     | id | app_id | coaching_org_id | coach_id | contact_id | team_number | start_date | student_names |
     | 1  | 1      | 1               | 1        | 2          | 1           | 2017-03-25 | s1, s2, s3    |
     | 2  | 1      | 1               | 1        | 2          | 2           | 2017-08-25 | s4, s5, s6    |
-
   And the following iterations exist:
     | id | engagement_id | end_date   |
     | 1  | 1             | 2017-04-14 |
     | 2  | 1             | 2017-04-28 |
+  And I'm logged in on the orgs page
+  And I follow "Apps"
+  And I follow "app1"
   
 Scenario: Can edit and destroy all existing engagements
   #Story ID: #152689950
-  Then I should see "Team1" has button "Edit"
-  And I should see "Team2" has button "Edit"
-  And I should see "Team1" has button "Destroy"
-  And I should see "Team2" has button "Destroy"
+  Then I should see "2017-03-25" has button "Edit"
+  And I should see "2017-08-25" has button "Edit"
+  And I should see "2017-03-25" has button "Destroy"
+  And I should see "2017-08-25" has button "Destroy"
+
+Scenario: Editing existing engagements is successful
+  #Story ID: #152689950
+  And I press "Edit" for "2017-03-25"
+  Then I should see "Editing Engagement for"
+  And I press "Save"
```

Comment:
I'm not sure if this was tested in other cucumber tests, but maybe we can add actual steps to edit the engagement, then check to see of those changes were actually saved

Reply:
That's a good suggestion. I initially had steps editing the engagement to see if it is working but then saw engagement_users.feauture already does that so I deleted them. Same applies for editing an iteration.